reputation/reputation.go

```diff
@@ -252,7 +252,7 @@ func calculateScore(addr string, mi miner.IndexSnapshot, si faults.IndexSnapshot
 		askScore = 1
 	}
 
-	score := 50*faultsScore + 20*powerScore + 20*externalScore + 10*askScore
+	score := 50*faultsScore + 20*powerScore + 20*externalScore + 10*askScore + 1000*float64(miner.ActiveDeals)
```

Comment:
Now that we have the number of active deals per miner, use it to bias heavily on that in the reputation module. That ratio is somewhat arbitrary as the others, but for testnet looks like the best one we can rely on for miner selection.